CONTRIBUTING.md

```diff
@@ -76,7 +76,7 @@ And make sure the PR haven't been published before!
 
 There isn't (yet) a formal style guide for Inferno, so please take care to adhere to existing conventions:
 
-* Tabs, not spaces!
+* 2-space indentation, not tabs!
```

Comment:
Why did you change this for? Haha. @Havunen

Reply:
Because the contributor file stated tabs, but throughout the project are 2-spaces, and the .editorconfig is set to 2-spaces.

Reply:
Seems like a bug and something that Prettier should be enforcing on the project. When I started Inferno off 3 years ago, it was all tabs. Up to @Havunen what he feels should be the standard now though.

Reply:
Seems like prettier uses spaces so lets go with that, otherwise we need to change all files ;)